pkg/reconciler/taskrun/resources/apply.go

```diff
@@ -130,36 +130,36 @@ func paramsFromTaskRun(ctx context.Context, tr *v1beta1.TaskRun) (map[string]str
 
 // ApplyResources applies the substitution from values in resources which are referenced in spec as subitems
 // of the replacementStr.
-func ApplyResources(spec *v1beta1.TaskSpec, resolvedResources map[string]v1beta1.PipelineResourceInterface, replacementStr string) *v1beta1.TaskSpec {
+func ApplyResources(spec *v1.TaskSpec, resolvedResources map[string]v1.PipelineResourceInterface, replacementStr string) *v1.TaskSpec {
 	replacements := map[string]string{}
 	for name, r := range resolvedResources {
 		for k, v := range r.Replacements() {
 			replacements[fmt.Sprintf("resources.%s.%s.%s", replacementStr, name, k)] = v
-			// FIXME(vdemeester) Remove that with deprecating v1beta1
+			// FIXME(vdemeester) Remove that with deprecating v1
 			replacements[fmt.Sprintf("%s.resources.%s.%s", replacementStr, name, k)] = v
 		}
 	}
 
 	// We always add replacements for 'path'
 	if spec.Resources != nil && spec.Resources.Inputs != nil {
 		for _, r := range spec.Resources.Inputs {
-			replacements[fmt.Sprintf("resources.inputs.%s.path", r.Name)] = v1beta1.InputResourcePath(r.ResourceDeclaration)
-			// FIXME(vdemeester) Remove that with deprecating v1beta1
-			replacements[fmt.Sprintf("inputs.resources.%s.path", r.Name)] = v1beta1.InputResourcePath(r.ResourceDeclaration)
+			replacements[fmt.Sprintf("resources.inputs.%s.path", r.Name)] = v1.InputResourcePath(r.ResourceDeclaration)
+			// FIXME(vdemeester) Remove that with deprecating v1
```

Comment:
this comment should be left intact. We shouldn't have `InputResourcePath` in v1. Instead, pipelineresources here would need to be deserialized from v1 annotations.

Reply:
Thanks for the comments Lee. 

> Any reason to convert both Task and TaskRun in the same PR? Just curious
This PR is made only to raise the questions on the conversion question. Will be putting up a PR on v1-release branch later. Also I would avoid opening branches redundantly in the future. Thanks for the pointers!